src/modules/Contribute/pages/service.tsx

```diff
@@ -151,8 +151,9 @@ const ServicePage = ({ documentTypes }: { documentTypes: string[] }) => {
         notify(
           'error',
           t(
-            'contribute:service_page.could_not_create_issue',
-            `We're truly sorry but the automatic creation of this new service failed, we will need you to send us an email`
+            'error.createissue',
```

Comment:
To me this is less good than before because it does not give enough context to the translator.

Knowing on which page the user is when seeing the message is crucial
This I would name the page in the key.
And if it's in the component I would go for `<namespace>:<component-name>.<key>`

So here `contribute:service_page.could_not_create_issue` lokked very fine to me

Reply:
Agree, my key name is absoluty an horror 🙀